fiftyone/utils/eval/regression.py

```diff
@@ -69,6 +70,8 @@ def evaluate_regressions(
         gt_field ("ground_truth"): the name of the field containing the
             ground truth :class:`fiftyone.core.labels.Regression` instances
         eval_key (None): a string key to use to refer to this evaluation
+        eval_metrics (None): a list of tuples of ``fiftyone.operators.Operator``
```

Comment:
I dont have full context, but if you’re thinking of this being a custom plugin, calling it outside the plugin framework here will likely cause circular dependencies